plugins/services/src/js/containers/volume-detail/VolumeDetail.js

```diff
@@ -54,6 +54,10 @@ class VolumeDetail extends React.Component {
       />
     );
 
+    const sizeLabel = volume.type === 'External'
```

Comment:
Please use a proper condition or private util to get the respective size label instead of a ternary to improve readability.